spec/support/shared_examples/behaves_like_invitable.rb

```diff
@@ -0,0 +1,95 @@
+# shared examples for Invitable
+# Example usage:
+#  include_examples "Invitable", :workship_invitation, :workshop
+#
+RSpec.shared_examples 'Invitable' do |invitation_type, invitable_type|
+  let(:invitable) { Fabricate(invitable_type) }
+
+  context '#attendances' do
+    it 'permits accepted' do
+      invitation = Fabricate(invitation_type,
+                             invitable_type => invitable,
+                             attending: true)
+
+      expect(invitable.reload.attendances).to include(invitation)
+    end
+
+    it 'rejects un-accepted' do
```

Comment:
Can we change this to read non accepted. I find un-accepted a bit confusing.

Reply:
done! changed all un- to non-  I think the test pairings looked better 👍